src/cgmemmgr.cpp

```diff
@@ -141,13 +141,18 @@ static intptr_t init_shared_map()
 static void *alloc_shared_page(size_t size, size_t *id, bool exec)
 {
     assert(size % jl_page_size == 0);
-    auto file_mode = exec ? PAGE_EXECUTE_READWRITE : PAGE_READWRITE;
+    long unsigned int file_mode = exec ? PAGE_EXECUTE_READWRITE : PAGE_READWRITE;
```

Comment:
why `long unsigned int` instead of `uint32_t`? that first one just looks confusing.